src/Task/Task.php

```diff
@@ -126,12 +126,17 @@ public function isOnce()
     }
 
     /**
-     * @param array $servers
+     * @param array|string|null $servers
      * @return $this
      */
-    public function onlyOn($servers)
+    public function onlyOn($servers = null)
     {
-        $this->onlyOn = array_flip($servers);
+        if ($servers === null) {
+            $this->onlyOn = [];
+        } else {
+            $this->onlyOn = array_flip(is_array($servers) ? $servers : [$servers]);
```

Comment:
Yep, I was actually suggesting that bellow and was waiting for more responses. Thank you sir anyway